less/joust.less

```diff
@@ -65,6 +65,15 @@
 	transition: background 0.8s;
 }
 
+.player.inactive {
+	.short, .tall {
+		-webkit-filter: grayscale(75%) blur(3px);
+		-moz-filter: grayscale(75%) blur(3px);
+		-o-filter: grayscale(75%) blur(3px);
+		filter: grayscale(75%) blur(3px);
```

Comment:
Prefixing is done automatically by `gulp-autoprefixer` - just do the normal `filter`.